spec/requests/api/products_controller_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe "Api::Products", type: :request do
+  
+  describe "Create Product" do
+    let(:product_params) do
+      {
+        name: "test product",
+        code: "PROD01"
+      }
+    end
+
+    context "For Success" do
+      before do
+        post "/api/products/create", params: product_params
+      end
+      
+      it "returns http status success" do
+        expect(response).to have_http_status(201)
+      end
+
+      it "returns succees message" do
+        json_response = JSON.parse(response.body)
+        expect(json_response['message']).to eq("Product Created Successfully")
+      end
```

Comment:
Please add following specs as well:

1. It 'initializes new object of product' 
2. It 'saves product'